### PR TITLE
Bootstrap the 3.5-->3.6 upgrade playbook

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

aos-ci-test
[test]

### CODE REVIEW

**Comment by sdodson** [comment]
You should be running playbooks/byo/openshift-cluster/upgrades/v3_6/ rather than under common so you'll want to duplicate those too.

File: playbooks/byo/openshift-cluster/upgrades/v3_6/upgrade.yml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  tasks:
+  - include: ../../../../common/openshift-cluster/upgrades/cleanup_unused_images.yml
+
+- include: ../../../../common/openshift-cluster/upgrades/upgrade_control_plane.yml
+
+- include: ../../../../common/openshift-cluster/upgrades/upgrade_nodes.yml
+
+- include: ../../../../common/openshift-cluster/upgrades/post_control_plane.yml
+
+- include: ../../../../common/openshift-cluster/upgrades/v3_6/storage_upgrade.yml
```
Comment (stevekuznetsov) [inline playbooks/byo/openshift-cluster/upgrades/v3_6/upgrade.yml]: @sdodson is this still applicable? or is that migration only for 3.5

**Comment by stevekuznetsov** [comment]
@sdodson could you trigger the AOS CI tests please ... I don' think I'm on whatever list I need to be on

**Comment by sdodson** [comment]
aos-ci-test

**Comment by jhadvig** [comment]
@stevekuznetsov run the install_upgrade job with your changes and tey failed to upgrade the cluster. Here is the job output:
https://ci.openshift.redhat.com/jenkins/job/test_branch_origin_extended_conformance_install_update/93/console
I've also kept the machine. Name is `steve_upgrade_6` and put term. protection on it

**Comment by sdodson** [comment]
aos-ci-test

**Comment by stevekuznetsov** [comment]
Not sure that change to the comment format helped, rofl

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
[merge]

**Comment by stevekuznetsov** [comment]
@openshift-bot, the [last build](https://ci.openshift.redhat.com/jenkins/job/merge_pull_request_openshift_ansible/100/) failed from the following flakes:
 - [conformance](https://ci.openshift.redhat.com/jenkins/job/merge_pull_request_openshift_ansible/100/testReport/junit/(root)/Extended/_builds__Conformance__oc_new_app_should_succeed_with_a___name_of_58_characters/) failed on: https://github.com/openshift/origin/issues/13067

re[merge]

**Pull request merged**